solutions/swb-reference/src/services/dataSetService.ts

```diff
@@ -34,6 +34,8 @@ import { SwbAuthZSubject } from '../constants';
 import { getProjectAdminRole, getResearcherRole } from '../utils/roleUtils';
 import { Associable, DatabaseServicePlugin } from './databaseService';
 
+const timeToLiveSeconds: number = 60; // 1 min
```

Comment:
this seems a little to strict. I'm thinking `60 * 5` by be a better TTL or even as much as `60 * 15`. What you have offered here is 1 minute for a user to take an array of URLs an act on all of them before they expire.

Reply:
Will wait for SWB guidance on this one

Reply:
15 minutes sounds reasonable.

Reply:
updated.